core/src/core/pmmr.rs

```diff
@@ -940,6 +811,9 @@ pub fn bintree_postorder_height(num: u64) -> u64 {
 /// We know the positions of all leaves based on the postorder height of an MMR of any size
 /// (somewhat unintuitively but this is how the PMMR is "append only").
 pub fn is_leaf(pos: u64) -> bool {
+	if pos == 0 {
```

Comment:
this ugly border test case can be removed again when we change to 0-based positions in the future:-)